### PR TITLE
Update troubleshooting

### PR DESCRIPTION
**Pull request by joncojonathan**
Following a recent installation of LME I encountered a few problems.  Submitting in the hope these will help others.  Please let me know if you need me to make any changes and many thanks for LME.

### CODE REVIEW

**Review by duncan-ncc — APPROVED** [review]
Thanks for the PR, Good comments which should help people prior to the new release which resolves some of these automatically.

Also probably worth noting that we do not currently support DC logging, and that the log volumes could be significant from DC's and we would be hesitant to suggest something which could impact the functioning of a Domain Controller without further research.

I suppose this could be sorted with a statement along the lines of "NOTE: Log forwarding from Domain Controllers as part of LME is not officially supported and has not been tested or endorsed by NCSC. The contents of this blog may be helpful if this is your desired goal, but they may have unintended side effects and we will not support any issues that arise as a result of this"

**Comment by joncojonathan** [comment]
Thanks for the feedback @duncan-ncc.  I've added a disclaimer about DC logging, so please let me know if you're happy with that.  I can always remove the section altogether and just leave it with a note that DC logging is not supported if that's preferred.  Let me know?

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -16,3 +16,32 @@ Figure 1: Troubleshooting overview diagram
...
+Check the following:
+
+* Sysmon service is running on the client
+* The [LME-WEC-Client-GPO](https://github.com/ukncsc/lme/blob/master/Chapter%201%20Files/lme_gpo_for_windows.zip) is applying to the member server
+* That the member server has been rebooted to apply permissions to the logs ([see issue #41](https://github.com/ukncsc/lme/issues/41#issuecomment-554037796))
+
+### Events not forwarding from Domain Controllers
+Please be aware that Logging Made Easy does not currently support logging Domain Controllers, and the log volumes may be significant from servers with this role.  If you wish to proceed forwarding logs from your Domain Controllers please be aware you do this at your own risk!  Monitoring such servers has not been tested or endorsed by the NCSC and may have unintended side effects.
+
+If you wish to monitor Domain Controllers then the contents of [this blog post](https://rockyprogress.wordpress.com/2011/12/04/security-event-log-collection-from-a-domain-controller/) may be helpful.  Note the NCSC is not responsible for the content of external sites.
```
Comment (a-d-a-m-b) [inline docs/troubleshooting.md]: I would be more comfortable if this section is edited (line 30 removed). I think linking to 10 year old blog that we have no control over is not advised, and gives mixed messages.... "It's not supported, but look here" is not consistent. etc

**Review by a-d-a-m-b — CHANGES_REQUESTED** [review]
One minor change, but happy with the content. Thanks for the PR; great work :)

File: docs/troubleshooting.md (hunk 1 above)
Comment (joncojonathan) [reply docs/troubleshooting.md]: Hi @a-d-a-m-b, I've removed L30 - L33 as the reminder seemed superfluous given it was right next to the warning.  Valid points on referencing 10 year old content.

**Review by a-d-a-m-b — APPROVED** [review]
Thanks for the prompt edits (and apologies for my slow review!) :)

**Pull request merged**